### PR TITLE
Add (some) Homebrew guides and tutorials to add games

### PR DESCRIPTION
**Pull request by abUwUser**
This PR, as requested by 40aba15, it will add some Homebrew guides, as well as a tutorial (for the Wii atm) written by me. The tutorials may be replaced by guides if it is found reliable guides to do so.

As of now, the list is empty and have nowhere near items. Recommendations are always welcomy

Changes:
- Rename "Emulators" to "Consoles"

Additions:
- Homebrew section
- Guides
    - [x] Nintendo
      - [x] NES
      - [x] SNES
      - [x] N64
      - [x] GameCube
      - [x] Wii ([Wii Guide](https://wii.guide))
      - [x] 3DS ([Hacks.Guide](https://hacks.guide/))
      - [x] WiiU ([Hacks.Guide](https://hacks.guide/))
      - [x] Nintendo Switch ([Nintendo Homebrew's Nintendo Switch homebrew guide](https://nh-server.github.io/switch-guide/) | [noahc3's Nintendo Switch homebrew guide](https://switch.homebrew.guide/))
    - [ ] Sony
      - [x] PS1
      - [x] PS2
      - [x] PS3
      - [ ] PS4
      - [ ] PS5 <sup>(likely not existent)</sup>
      - [x] PSP
      - [x] PSVita ([Hacks.Guide](https://hacks.guide/))
    - [ ] Microsoft
      - [x] Xbox
      - [x] Xbox 360
      - [x] Xbox One
      - [ ] Xbox Series <sup>(likely not existent)</sup>
    - [ ] Sega
      - [x] Sega Master System
      - [ ] Sega Mega Drive
      - [ ] Sega Saturn
      - [x] Dreamcast
    - [x] Valve
      - [x] Steam Deck
    - [x] Other
        - [ConsoleMods](https://consolemods.org/)

- Piracy
    - [ ] Nintendo
        - [ ] NES
        - [ ] SNES
        - [ ] N64
        - [ ] GameCube
        - [x] Wii (written)
        - [ ] 3DS
        - [ ] WiiU
        - [ ] Nintendo Switch
    - [ ] Sony
        - [ ] PS1
        - [ ] PS2
        - [ ] PS3
        - [ ] PS4
        - [ ] PS5
        - [ ] PSP
        - [ ] PSVita
    - [ ] Microsoft
        - [ ] Xbox
        - [ ] Xbox 360
        - [ ] Xbox One
        - [ ] Xbox Series
    - [ ] Sega
        - [ ] Sega Master System
        - [ ] Sega Mega Drive
        - [ ] Sega Saturn
        - [ ] Dreamcast
    - [x] Valve
      - [x] Steam Deck ([Piracy Guides](https://gitlab.com/ZediAlreadyTaken/guides))

### CODE REVIEW

File: Consoles/Homebrew.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+[Hacks.Guide's PS Vita guide](https://vita.hacks.guide/) - Guide to homebrew the PS Vita made by the Hacks.Guide team
+
+### Other
+
+[ConsoleMods](https://consolemods.org/) - Community driven wiki dedicated for modding
+
+
+## Playing pirated games
+This section provides information about playing pirated games listed or written by ourselves, as the listed guides are against piracy.
```
Comment (Helmasko) [inline Consoles/Homebrew.md:28]: you should clarify that its written by you specifically

**Comment by Helmasko** [comment]
looks cool, ill have a deeper look later on

File: Consoles/Homebrew.md (hunk 1 above)
Comment (abUwUser) [reply Consoles/Homebrew.md:28]: done on 8c5d774

**Comment by abUwUser** [comment]
had a better look?

**Comment by Helmasko** [comment]
> had a better look?

no cuz i forgot to, also might have to wait a lil longer cause of irl stuff

**Comment by abUwUser** [comment]
oh okay. good luck anyways!

**Comment by abUwUser** [comment]
hello. are you okay to review now? this PR is now like 3 weeks old

**Comment by Helmasko** [comment]
The reason why it's taking time is because some collabs are not responding to my proposed ideas about this PR, I don't want to make big decisions just by myself (changing the name of the .md for example). And I think I'll be very busy next week so who knows when anything will happen. But you don't need to keep reminding cuz I am aware.

**Comment by abUwUser** [comment]
that's good. but did you meant my PR or this branch? (the `New_Emulators` branch)

**Comment by Helmasko** [comment]
ok ive done my first checks and have fixed some grammar and layout, looks good but i need to do another check later on.

**Comment by Helmasko** [comment]
i also merged emulators.md into ROMs.md cuz emulators would be too short if i just inlcuded the emugen link and nothing else.

**Comment by abUwUser** [comment]
do you think that there is any other guide for homebrewing consoles? or how to install pirated games on them? those are as far as i know of. if no, i guess it is better to mark as ready for review and merge it so another person could give it a read and suggest another guide

**Comment by Helmasko** [comment]
I plan on putting a few things myself, idk if ill put them in the PR or once on the site. dont worry about other collabs

**Comment by abUwUser** [comment]
it is better to put on the PR, so i could suggest changes over here, that is if you want to have that

**Comment by abUwUser** [comment]
Any news?
I am not much of a fan with PRs opened for that long :/

**Comment by Helmasko** [comment]
Work work work needs doing

currently, https://consolemods.org/wiki/Main_Page, https://emulation.gametechwiki.com/index.php/ROM_%26_ISO_sites needs to be checked. if we decide to add the first link then it might be pointless to include hacks.guide or the 3ds guide. if we include the second link the ROM downloads page may need to be revised from ground up (unlikely)
various subreddits for console piracy can be added like /r/SwitchPirates/ or r/VitaPiracy/ but we need to find the subreddits for the other consoles too.
unsure if we should include steam deck cuz it is really console like in terms of modifications etc. but if we include it do we also include PC gaming related tools too like protondb site, or just link to PC gaming section?
no one seems to be interested in helping out with the PR and im not really the console guy (one of the sections i didnt touch much)

so if its really ticking you off for having your PR wait, feel free to help out.

**Comment by abUwUser** [comment]
> if we decide to add the first link then it might be pointless to include hacks.guide or the 3ds guide

not quite. if i could do an analogy it would be like having wikipedia and a hacking wiki. wikipedia may contain a lot of information, but it isn't specialized on hacking, and so it does not include everything more in depth as that hacking wiki does. it is simmilar to consolemods (wikipedia) and the 3ds guide (the hacking wiki)

> various subreddits for console piracy can be added like /r/SwitchPirates/ or r/VitaPiracy/ but we need to find the subreddits for the other consoles too.

honestly, i am not a user of reddit, so i can't recommend anything, but i can list those on the page. maybe in a "Subreddits" or "Forums" section?

> unsure if we should include steam deck cuz it is really console like in terms of modifications etc. but if we include it do we also include PC gaming related tools too like protondb site, or just link to PC gaming section?

it would be good to add, but it will depend on what we'll add, because if it is for example a guide to steam deck, then maybe it will already link that out. but i don't know, because i don't see much else that i could add as informative for the steam deck other than protondb. i am not a steam deck user tho

**Comment by abUwUser** [comment]
also if we add the steam deck, would we link [Piracy Guides](https://gitlab.com/ZediAlreadyTaken/guides) to it?

**Comment by abUwUser** [comment]
Also i have some pending reviews. Is there anything that you want to comment on them?

**Comment by Krametos** [comment]
> not quite. if i could do an analogy it would be like having wikipedia and a hacking wiki. wikipedia may contain a lot of information, but it isn't specialized on hacking, and so it does not include everything more in depth as that hacking wiki does. it is somewhat simmilar to consolemods (wikipedia in that case) and the 3ds guide (the hacking wiki)

I think you misunderstood me, I meant remove 
> [Hacks.Guide](https://hacks.guide/) - Guides made by the Hacks.Guide team to Homebrew the 3ds, WiiU and PS Vita.

Because the same exact site is linked on consolemods.org which makes it redundant. Kinda how we're revamping the emulators section cuz emugen is just better in every way.
>honestly, i am not a user of reddit, so i can't recommend anything, but i can list those on the page. maybe in a "Subreddits" or "Forums" section?

I can manage the layout, we just need to find all of them.
>Also i have some pending reviews. Is there anything that you want to comment on them?

Can't see any pending reviews, tbh not even entirely sure how to check for that but I've looked at your commits I'll make a few amendments and then we'll see where it goes.

**Comment by Helmasko** [comment]
> not quite. if i could do an analogy it would be like having wikipedia and a hacking wiki. wikipedia may contain a lot of information, but it isn't specialized on hacking, and so it does not include everything more in depth as that hacking wiki does. it is somewhat simmilar to consolemods (wikipedia in that case) and the 3ds guide (the hacking wiki)

I think you misunderstood me, I meant remove 
> [Hacks.Guide](https://hacks.guide/) - Guides made by the Hacks.Guide team to Homebrew the 3ds, WiiU and PS Vita.

Because the same exact site is linked on consolemods.org which makes it redundant. Kinda how we're revamping the emulators section cuz emugen is just better in every way.
>honestly, i am not a user of reddit, so i can't recommend anything, but i can list those on the page. maybe in a "Subreddits" or "Forums" section?

I can manage the layout, we just need to find all of them.
>Also i have some pending reviews. Is there anything that you want to comment on them?

Can't see any pending reviews, tbh not even entirely sure how to check for that but I've looked at your commits I'll make a few amendments and then we'll see where it goes.

File: Consoles/Homebrew.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+[Homebrew Guide](https://switch.homebrew.guide/) - Another Nintendo Switch Homebrew guide. This guide uses SDSetup, tool that is not recommended by some members, but other than that this guide is fine.
+
+### Sony
+
+[Vita Guide](https://vita.hacks.guide/) - Guide to Homebrew the PS Vita made by the Hacks.Guide team.
+
+### Other
+
+[ConsoleMods](https://consolemods.org/) - Community driven Wiki dedicated for modding.
```
Comment (Helmasko) [inline Consoles/Homebrew.md:24]: have u checked this site already? cuz the reason i didnt add it was i hadnt checked its legitamacy

**Comment by Helmasko** [comment]
sites that need checking
https://www.reddit.com/r/ps3homebrew/
https://www.reddit.com/r/originalxbox/
https://www.reddit.com/r/360hacks/
https://www.reddit.com/r/romhacking/
https://www.romhacking.net/  -  https://datacrystal.romhacking.net/wiki/Introduction_to_Hacking
we could also add this https://gitlab.com/ZediAlreadyTaken/guides/-/blob/main/consoles.md but may be redundant depending on what we have

**Comment by abUwUser** [comment]
> Because the same exact site is linked on consolemods.org which makes it redundant. Kinda how we're revamping the emulators section cuz emugen is just better in every way.

i don't trust consolemods.org that much, since it is incomplete in some parts, and it is more of a wiki than guides. it is more of a source to find some specific information about a console, as i said, like a "wikipedia" of console internals, it isn't specific as a proper wiki for a specific console, but it gets the job done

**Comment by abUwUser** [comment]
> Can't see any pending reviews, tbh not even entirely sure how to check for that but I've looked at your commits I'll make a few amendments and then we'll see where it goes.

Screenshot_20230305_212541_Kiwi Browser

**Comment by abUwUser** [comment]
> we could also add this https://gitlab.com/ZediAlreadyTaken/guides/-/blob/main/consoles.md but may be redundant depending on what we have

doesn't have that much info to be honest

**Comment by Helmasko** [comment]
> > Can't see any pending reviews, tbh not even entirely sure how to check for that but I've looked at your commits I'll make a few amendments and then we'll see where it goes.
Since I can't locate those reviews I'll just comment on them here.
1-2. The way we pick the names are almost always what you see on the searchbar when you put the link in, but you bring up a good point that it makes it difficult to see which console its a guide for. I'll see if I can find a better solution and still keep the names concise and close to the original.
3. because your guide will work for non-pirated, legitemate copies too, (i hope).

---

About your analogy of wikipedia and consolemods i still dont get it. All I'm saying is as a small example, if we include consolemods, including https://hacks.guide/ would be redundant. before you say the latter is more detailed. consolemods already links to hacks.guide. on top of that it also links to other resources like https://wii.guide/ and nicely divides the consoles into different sections with images so you can easily find a **detailed guide** for a specific console. Its difficult going back and forth on the same topic here, if u still wanna talk about this lets continue this on discord.

**Comment by Helmasko** [comment]
consolemods doesnt include newest gen consoles, and we will keep your guide so it wont be empty

**Comment by Helmasko** [comment]
ive also noticed there are multiple guides per consoles, e.g. https://wiibrew.org/wiki/Main_Page, wii.guide and consolemodes' own guide, so i will include all of them unless they look outdated

**Comment by abUwUser** [comment]
so okay. at the end we will just, keep both consolemods and per console guides?

**Comment by Helmasko** [comment]
> so okay. at the end we will just, keep both consolemods and per console guides?

when there is no overlap, im in the process of doing it atm.

**Comment by abUwUser** [comment]
wdym by overlap?

**Comment by Helmasko** [comment]
> wdym by overlap?

when they dont link to the same guide, like removing vita.hacks.guide cuz its already linked in consolemods but keeping wii.guide cuz its not.

**Comment by Helmasko** [comment]
we also need to find guides for current gen consoles, we already got switch, but ps5, xbox series?

**Comment by abUwUser** [comment]
> > wdym by overlap?
> 
> when they dont link to the same guide, like removing vita.hacks.guide cuz its already linked in consolemods but keeping wii.guide cuz its not.

understandable 

> we also need to find guides for current gen consoles, we already got switch, but ps5, xbox series?

i do not know about the ps5 and xbox series homebrewing community, but i know that yhe xbox one / series has a developer mode where UWP apps can be loaded. other than that i have no idea about the rest

**Comment by Helmasko** [comment]
ok so far were almost done, currently https://www.romhacking.net/ needs to be checked, and the piracy subreddits like vitapiracy, switchpiracy needs to be added (ive added the homebrew piracy subreddits). feel free to help out

**Comment by abUwUser** [comment]
may i close this PR due to the addition of https://ripped.guide/Consoles/Homebrew-WIP/ ?

**Comment by abUwUser** [comment]
> may i close this PR due to the addition of https://ripped.guide/Consoles/Homebrew-WIP/ ?

Nevermind, apparently that page links to this PR. Anyways, is it updated manually the page? Or it automatically changes when a commit is made here?

**Comment by abUwUser** [comment]
> currently, https://consolemods.org/wiki/Main_Page, https://emulation.gametechwiki.com/index.php/ROM_%26_ISO_sites needs to be checked. if we decide to add the first link then it might be pointless to include hacks.guide or the 3ds guide. if we include the second link the ROM downloads page may need to be revised from ground up (unlikely)

Checking it better, it is better to include 3ds.hacks.guide, mainly because it is a very trusted guide for 3ds homebrewing

**Comment by unclamped** [comment]
for switch, bnN3MnU= is usually kewl

**Comment by kekkodance** [comment]
we already have it in the guide?

**Comment by unclamped** [comment]
oh, my bad. i forgot this was about homebrew and not games 😬

**Comment by abUwUser** [comment]
honestly, do you recommend any guides to put in?

**Comment by Helmasko** [comment]
ok i've decided to add the 'overlapping links' since the page looked a little too bare, added a few more steam deck tools. rn looking at a few more general rom sites and then we should be good to go.

**Comment by Helmasko** [comment]
Anything we should do about the spacing between lines, it seems inconsistent? Also nows the time for final touches if you hve anything, I'll me merging this soon as it's been here long enough

**Pull request merged**